<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[launching FireFoxBrowser]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-25T12:02:13 IST" name="Default suite" finished-at="2023-07-25T12:02:35 IST" duration-ms="22365">
    <groups>
    </groups>
    <test started-at="2023-07-25T12:02:13 IST" name="Default test" finished-at="2023-07-25T12:02:35 IST" duration-ms="22365">
      <class name="com.flipkart.qa.test.FK_TC4400_ValidateUserName">
        <test-method is-config="true" signature="openingBrowser()[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:13 IST" name="openingBrowser" finished-at="2023-07-25T12:02:35 IST" duration-ms="22163" status="FAIL">
          <exception class="org.openqa.selenium.remote.NoSuchDriverException">
            <message>
              <![CDATA[Unable to obtain: Capabilities {acceptInsecureCerts: true, browserName: firefox, moz:debuggerAddress: true, moz:firefoxOptions: {}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.NoSuchDriverException: Unable to obtain: Capabilities {acceptInsecureCerts: true, browserName: firefox, moz:debuggerAddress: true, moz:firefoxOptions: {}}
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:29)
at org.openqa.selenium.firefox.FirefoxDriver.generateExecutor(FirefoxDriver.java:139)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:130)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:125)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:110)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:100)
at com.flipkart.qa.base.BaseClass.lanuchBrowser(BaseClass.java:34)
at com.flipkart.qa.test.FK_TC4400_ValidateUserName.openingBrowser(FK_TC4400_ValidateUserName.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.WebDriverException: Unsuccessful command executed: [C:\Users\CHINTU\AppData\Local\Temp\selenium-manager2398630760366619613489036860395876722\selenium-manager.exe, --browser, firefox, --output, json]
error sending request for url (https://github.com/mozilla/geckodriver/releases/latest): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '18.0.1'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.manager.SeleniumManager.runCommand(SeleniumManager.java:130)
at org.openqa.selenium.manager.SeleniumManager.getDriverPath(SeleniumManager.java:251)
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:27)
... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.NoSuchDriverException -->
          <reporter-output>
            <line>
              <![CDATA[launching FireFoxBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- openingBrowser -->
        <test-method signature="lanuchBrowser(java.lang.String)[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:35 IST" name="lanuchBrowser" finished-at="2023-07-25T12:02:35 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'bName' is required by @Test on method lanuchBrowser but has not been marked @Optional or defined
in C:\Users\CHINTU\AppData\Local\Temp\testng-eclipse--531827943\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'bName' is required by @Test on method lanuchBrowser but has not been marked @Optional or defined
in C:\Users\CHINTU\AppData\Local\Temp\testng-eclipse--531827943\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:270)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:358)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.handleParameters(Parameters.java:884)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lanuchBrowser -->
        <test-method is-config="true" signature="logIn()[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:35 IST" name="logIn" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1690266755466" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method signature="validateUserName()[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:35 IST" name="validateUserName" finished-at="2023-07-25T12:02:35 IST" duration-ms="0" status="SKIP">
          <exception class="org.openqa.selenium.remote.NoSuchDriverException">
            <message>
              <![CDATA[Unable to obtain: Capabilities {acceptInsecureCerts: true, browserName: firefox, moz:debuggerAddress: true, moz:firefoxOptions: {}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.NoSuchDriverException: Unable to obtain: Capabilities {acceptInsecureCerts: true, browserName: firefox, moz:debuggerAddress: true, moz:firefoxOptions: {}}
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:29)
at org.openqa.selenium.firefox.FirefoxDriver.generateExecutor(FirefoxDriver.java:139)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:130)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:125)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:110)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:100)
at com.flipkart.qa.base.BaseClass.lanuchBrowser(BaseClass.java:34)
at com.flipkart.qa.test.FK_TC4400_ValidateUserName.openingBrowser(FK_TC4400_ValidateUserName.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.WebDriverException: Unsuccessful command executed: [C:\Users\CHINTU\AppData\Local\Temp\selenium-manager2398630760366619613489036860395876722\selenium-manager.exe, --browser, firefox, --output, json]
error sending request for url (https://github.com/mozilla/geckodriver/releases/latest): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '18.0.1'
Driver info: driver.version: FirefoxDriver
at org.openqa.selenium.manager.SeleniumManager.runCommand(SeleniumManager.java:130)
at org.openqa.selenium.manager.SeleniumManager.getDriverPath(SeleniumManager.java:251)
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:27)
... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.NoSuchDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserName -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:35 IST" name="logout" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1690266755495" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.flipkart.qa.test.FK_TC4400_ValidateUserName@209da20d]" started-at="2023-07-25T12:02:35 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1690266755507" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.flipkart.qa.test.FK_TC4400_ValidateUserName -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
